docs/install-with-kind-k8s.md

```diff
@@ -0,0 +1,93 @@
+- create kind cluster
+
+```
+cat <<EOF | kind create cluster --name mine --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+        authorization-mode: "AlwaysAllow"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+- role: worker
+- role: worker
+- role: worker
+EOF
+```
+
+- install go  gofmt  golangci-lint  kube-apiserver  kubebuilder  kubectl  kustomize  pkger etcd
+```
+export PATH=~/go/bin:$PATH
+```
+
+- install cert-manager
+```bash
+helm repo add jetstack https://charts.jetstack.io
+
+helm repo update
+
+kubectl create namespace cert-manager
+
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v0.13.1
+```
+
+- install harbor-operator-system
+```bash
+kubectl create namespace harbor-operator-system
+make deploy
+```
+
+- install nginx-ingess with nodeport
+```bash
+helm install nginx stable/nginx-ingress
+
+kubectl patch deployments nginx-nginx-ingress-controller -p '{"spec":{"template":{"spec":{"containers":[{"name":"nginx-ingress-controller","ports":[{"containerPort":80,"hostPort":80},{"containerPort":443,"hostPort":443}]}],"nodeSelector":{"ingress-ready":"true"},"tolerations":[{"key":"node-role.kubernetes.io/master","operator":"Equal","effect":"NoSchedule"}]}}}}'
```

Comment:
I suggested you changes about that awful patch: https://github.com/goharbor/harbor-operator/pull/27/commits/9fa9bffe412c1df6883101eb1a0a969c43bd2027
Please reuse it.

Reply:
not work yet